geniza/corpus/admin.py

```diff
@@ -63,7 +63,7 @@ class TextBlockInline(admin.TabularInline):
     autocomplete_fields = ['fragment']
     readonly_fields = ('thumbnail', )
     fields = ('fragment', 'side', 'extent_label', 'multifragment',
-              'order', 'thumbnail')
+              'order', 'thumbnail', 'certain')
```

Comment:
I think display before the thumbnail (input fields before display/context fields)